cinch/library/jenkins_plugin.py

```diff
@@ -0,0 +1,826 @@
+#!/usr/bin/python
```

Comment:
There were two bugs that caused the module to error out installing some of the plugins. I've modified them in this version, and I'll be putting in a PR against upstream if they haven't been resolved there, already.